pkg/project/project.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
```

Comment:
Is it possible to passthrough the **logging.Logger** instance from:

- `internal/commands/build.go` to `func parseProjectToml(appPath, descriptorPath string)` and finally to `func ReadProjectDescriptor(pathToFile string)` in that way we don't need to use `fmt`
 

func ReadProjectDescriptor(pathToFile string) (types.Descriptor, error)

Reply:
@jjbustamante thanks for that suggestion. By passing through the logger I was able to drop the more expensive acceptance tests and replace them with equivalent unit tests.